scripts/show-capabilities-by-function.py

```diff
@@ -138,6 +138,14 @@ def main(argv=None):
             default="auto",
             help="Select sample format, %s" % format_help,
         )
+        parser.add_argument(
```

Comment:
maybe it's worth creating a function that sets up the share arguments so we don't have to update all the scripts all the time?

Reply:
yeah, i was feeling the same way, since i needed to make changes to like six files. i'll take a stab in another branch.